.gitlab-ci/stages/deploy.yml

```diff
@@ -142,8 +142,6 @@ Deploy @cdtn/frontend (dev):
   after_script:
     - |-
       source ./.gitlab-ci/env.sh
-      echo "Frontend url is : ${FRONTEND_URL}";
-      echo "Last commit is : ${COMMIT}";
```

Comment:
domage, c'était bien de les laisser (au moins commit)

Reply:
Le commi il est echo dans chaque etape de stage un peu plus haut is tu en as besoin ! (lors de l'exec de env.sh) et il est re echo lorsque l'api front repond avec un match